hail/python/hail/docs/_templates/_autosummary/class2.rst

```diff
@@ -4,7 +4,6 @@
 
 .. autoclass:: {{ objname }}()
     :members:
-    :show-inheritance:
```

Comment:
This is a useful feature, but because our docs document things in a place *other than where they are defined* (e.g. CollectionExpression's docs are at hail.expr.CollectionExpression instead of hail.expr.expressions.typed_expressions.CollectionExpression`) autodoc simply does not work (it cannot resolve the reference).